src/core/plugins/spec/selectors.js

```diff
@@ -494,12 +494,21 @@ export const validationErrors = (state, pathMethod) => {
   let paramValues = state.getIn(["meta", "paths", ...pathMethod, "parameters"], fromJS([]))
   const result = []
 
+  const formatErrors = (errors) => {
```

Comment:
I'm not sure about the names and placement of these functions.

Reply:
Placement and names of the functions is just fine.